src/main/scripts/src/app/shared/perspectives/documents/table/table-documents-perspective.component.ts

```diff
@@ -84,24 +84,26 @@ export class TableDocumentsPerspectiveComponent implements Perspective, OnInit {
 
   public onNewValue(dataEvent: DataEvent) {
     this.documentService.createDocument(this.collection.code, this.convertDataEventToDocument(dataEvent))
-      .subscribe((json: object) => this.rows[dataEvent.rowIndex].id = json['_id']);
+      .subscribe(res => console.log(res));
   }
 
   public onValueChange(dataEvent: DataEvent) {
-    this.documentService.updateDocument(this.collection.code, this.convertDataEventToDocument(dataEvent))
+    this.documentService.patchDocument(this.collection.code, this.convertDataEventToDocument(dataEvent))
       .subscribe();
   }
 
   public onHeaderChange(dataEvent: DataEvent) {
-    let oldValue: string = dataEvent.data.oldValue;
-    let newValue: string = dataEvent.data.newValue;
-    this.collectionService.renameAttribute(this.collection.code, oldValue, newValue)
-      .subscribe();
+    // TODO
+    // let oldValue: string = dataEvent.data.oldValue;
+    // let newValue: string = dataEvent.data.newValue;
+    // this.collectionService.renameAttribute(this.collection.code, oldValue, newValue)
+    //   .subscribe();
```

Comment:
It should be possible to work with attributes now. However, the structure is slightly different.